### ISSUE TITLE
Number#toPrecision fails in Opera 10.6

### ISSUE THREAD

**Issue by ljharb** [es-shims/es5-shim#366]
Fails with `Expected '0.00008' to be '0.0000800'.`

**Comment by Xotic750** [comment]
![bs_winxp_opera_10 6](https://cloud.githubusercontent.com/assets/216041/12643290/8c9025fa-c5bb-11e5-97d4-56262f64a2c7.jpg)

**Comment by ljharb** [comment]
thanks, this bug is solely about Number#toPrecision

**Comment by Xotic750** [comment]
Yup, was just posting the screenshot as I managed to get on 10.6 ;)
